required/latex-lab/latex-lab-amsmath.dtx

```diff
@@ -177,39 +788,41 @@
            \tag_suspend:n{\text@}
            \mathchoice
             {
-             \@@_tag_if_mathstyle:en{mathchoice-\int_use:N\g__math_mathchoice_int}{0}
+             \__math_tag_if_mathstyle:en{mathchoice-\int_use:N\g__math_mathchoice_int}{0}
```

Comment:
Why not `@@` here and in `\g__math_mathchoice_int` (same for following lines)

Reply:
Because I disabled the tag replacement in latex-lab-amsmath. When you copy lots of internals from amsmath with lots of double @@ in the command names, tag replacements gets very error prone.